plugins/src/webrtcsink/imp.rs

```diff
@@ -2105,7 +2161,7 @@ impl ObjectSubclass for WebRTCSink {
     const NAME: &'static str = "RsWebRTCSink";
     type Type = super::WebRTCSink;
     type ParentType = gst::Bin;
-    type Interfaces = (gst::ChildProxy,);
+    type Interfaces = (gst::ChildProxy, gst_video::Navigation,);
```

Comment:
```suggestion
    type Interfaces = (gst::ChildProxy, gst_video::Navigation);
```

The trailing comma is only need for 1-tuples to distinguish them from a plain value